QuickOSM/ui/base_processing_panel.py

```diff
@@ -98,7 +98,7 @@ def _end_process(self):
         if self.panel == Panels.Query:
             self.dialog.button_generate_query.setDisabled(False)
 
-        self.dialog.output_directories[self.panel].setDisabled(True)
+        self.dialog.output_directories[self.panel].setDisabled(False)
```

Comment:
If you don't this here, you need to update the "dev" branch with this commit which is already in master